resources/views/home/_inventory_stack.blade.php

```diff
@@ -93,7 +93,7 @@
                         </div>
                     </li>
                 @endif
-                @if($item->canDonate)
+                @if(isset($item->category) && $item->canDonate)
```

Comment:
Lookin' in the wrong place here-- since this would only ever allow items with a category to be donated, where that's not explicitly the intent. Rather, consider that attribute, `canDonate`: there are several places in which it assume a category exists. So, a fun trick you can do-- at least in recent versions of Laravel, unsure how far back it goes-- is follow getting a relation with a `?` if it may or may not exist... e.g. in this case `$item->category?->attribute`. Might try that on that attribute instead, or, failing that, `$this->category && $this->category->etcetc`.